com.unity.perception/Documentation~/Randomization/Scenarios.md

```diff
@@ -1,32 +1,46 @@
 # Scenarios
 
 Scenarios have three responsibilities:
-1. Controlling the execution flow of your simulation 
-2. Customizing the application of random parameters in your project
-3. Defining constants that can be configured externally from a built Unity player
+1. Controlling the execution flow of your simulation
+2. Defining a list of randomizers
+3. Defining constants that can be configured externally from a built Unity player 
 
-By default, the perception package includes one ready-made scenario, the `FixedFrameLengthScenario` class. This scenario is useful for when all created parameters have target GameObjects configured directly in the `ParameterConfiguration` and the scenario execution requires little modification.
+By default, the perception package includes one ready-made scenario, the `FixedLengthScenario` class. This scenario runs each iteration for a fixed number of frames and is compatible with the Run in USim window for cloud simulation execution.
```

Comment:
Great point, though I'm not sure if I know of such a hook between documentation and scripts.

Reply:
Since we don't quite have an auto update mechanism to address this issue, I think I will mark this item as resolved and note that we will need to manually peruse our docs before future releases to catch class name changes like this.